config.ini

```diff
@@ -8,6 +8,11 @@ NumberOfTemplates=0
 # Which DTU to be used ahoy, opendtu, template
 DTU=opendtu
 
+# If you want to exclude Inverter, specify how many of the Inverters you want to query.
```

Comment:
I appreciate your comment here. Could an example of what a user might choose help here?  I mean, I'm a developer, but it's hard for me to imagine what row of data is meant here. Also, does it affect both types of controllers? Ahoy and DTU, or just one?